docs/tctl/admin/db/clean.md

```diff
@@ -89,7 +89,7 @@ The number of shards handled between each emittance of progress.
 
 Default: 10
 
-::: note
+:::note
 Enable `--tls` before using any of the modifiers below.
 :::
```

Comment:
```suggestion
:::note

Enable `--tls` before using any of the following modifiers.

:::
```
Prettier might've made the changes, per https://docusaurus.io/docs/markdown-features/admonitions#usage-with-prettier. Adding LFs to avoid future Prettier changes.